plugins/flashrom/README.md

```diff
@@ -17,9 +17,36 @@ This plugin supports the following protocol ID:
 
  * org.flashrom
 
+BIOS Version String
```

Comment:
Probably need to prefix this with Coreboot

Reply:
Fixed.